internal/internal.go

```diff
@@ -238,6 +238,11 @@ var (
 	// SetBufferPoolingThresholdForTesting updates the buffer pooling threshold, for
 	// testing purposes.
 	SetBufferPoolingThresholdForTesting any // func(int)
+
+	// HTTPSProxyFromEnvironment returns the URL of the proxy to use
+	// for testing purposes. It is used to override the `http.ProxyFromEnvironment`
+	// function for testing purposes.
+	HTTPSProxyFromEnvironment any //func(*http.Request) (*url.URL, error)
```

Comment:
Nit: space after slashes for consistency..

But also: why bother with making it an `any`?  It can just have the proper type.  The reason for the `any`s is to avoid circular dependencies, but the http and url packages in OSS will never import grpc/internal.  :)

Or even, let's just move this directly into internal/resolver/delegatingresolver as an exported thing there since it won't need to be changed by any non-grpc code.

Reply:
I asked to keep it `any` to limit the number of packages that `internal` imports.

Reply:
Done.